api-controllers/ServiceFabrikApiController.js

```diff
@@ -482,7 +482,7 @@ class ServiceFabrikApiController extends FabrikBaseController {
           return this.backupStore.getBackupFile(backupFileOptions);
         }
       })
-      .catchThrow(NotFound, new UnprocessableEntity(`Cannot restore for guid/timeStamp '${timeStamp || backupGuid}' as no successful backup found in this space`))
+      .catchThrow(NotFound, new UnprocessableEntity(`Cannot restore for guid/timeStamp '${timeStamp || backupGuid}' as no successful backup found in this space.`))
```

Comment:
comment is related to LN 490 - shouldn't we also pass source instance id and ensure they belong to the same plan as the instance id being restored or  check the restore predecessors & see if its allowed?

Reply:
The restore predecessor is check in LN 490. As the input is plan id of metadata of backup, and checked with current instance's plan or whether it is predecessor of current plan. So, I think by that it is already covered.